schema/properties_directory/non-scalar/file.json

```diff
@@ -0,0 +1,27 @@
+{
+    "schemaId": "properties-directory-file",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "file property schema",
+    "properties": {
+        "name": {
```

Comment:
This has to be an enum equal to `file_content`, perhaps, and we need to add other keywords:
pathname, basename - how does that sound?

Reply:
It might make sense to define the pathname/basename schema separately and include it to resolve https://github.com/Exabyte-io/esse/pull/164/files#r586772179 as well

Reply:
Property name has been defined and set to `file_to_display` per our conversation earlier this afternoon.

Recent commits also extract the pathname/basename into a file_metadata schema, under `core/reusable`